### PR TITLE
#feat - project setup with mockbee and authntication pages created.

### PR DESCRIPTION
**Pull request by surajmore98**
- project created using mockbee structure.
- added images and product data.
- added routes
- added navbar, login, logout and register component
- added mock component for api

### CODE REVIEW

File: src/Mock.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import Mockman from "mockman-js";
+
+function Mock() {
+  return (
+    <div className="MockAPI">
+      <Mockman />
+    </div>
+  );
+}
+
```
Comment (lomesshh) [inline src/Mock.js]: you have 2 similar components ( src/mock & component/mock ) with the same functionality remove any one of them

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <div className="App">
+      <Routes>
+          <Route path="/" element={<Home/>} /> 
+          <Route path="/login" element={<Login/>} />
+          <Route path="/register" element={<Register/>} />
+          <Route path="/logout" element={<Logout/>} />
+          <Route path="/mock" element={<Mock/>} /> 
+      </Routes>
+      {/* <ScriptTag type="text/javascript" 
+      src="http://127.0.0.1:5500/src/js/index.js" /> */}
```
Comment (lomesshh) [inline src/App.js]: remove this comment

File: src/component/home/Home.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function Home() {
+
+}
+
+export default Home;
```
Comment (lomesshh) [inline src/component/home/Home.js:5]: remove this empty component

**Review by lomesshh — APPROVED** [review]
do these suggested changes 
all rest looks good to me

**Review by samyakshah3008 — APPROVED** [review]
Small suggestion: 
Please make small PRs as this is too large for good review

**Review by MurtazaBagwala07 — APPROVED** [review]
I like the folder structure
Code is also clean and readable

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <div className="App">
+      <Routes>
+          <Route path="/" element={<Home/>} /> 
+          <Route path="/login" element={<Login/>} />
+          <Route path="/register" element={<Register/>} />
+          <Route path="/logout" element={<Logout/>} />
+          <Route path="/mock" element={<Mock/>} /> 
+      </Routes>
+      {/* <ScriptTag type="text/javascript"
```
Comment (Rohanmond) [inline src/App.js]: remove these comments

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import { makeServer } from "./server";
+import { BrowserRouter as Router } from "react-router-dom";
+//import { ProductProvider } from "./provider/ProductProvider";
```
Comment (Rohanmond) [inline src/index.js]: remove these comments

**Review by Rohanmond — APPROVED** [review]

File: src/index.js (hunk 5 above)
Comment (surajmore98) [reply src/index.js]: done

File: src/component/home/Home.js (hunk 3 above)
Comment (surajmore98) [reply src/component/home/Home.js:5]: just added for next PR.

**Pull request merged**